### ISSUE TITLE
a11y: Navbar - Help: cannot highlight subTitle

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#52841]
I often want to copy/paste the grafana version / commit hash from the HELP nav-bar, but it's not focusable:

https://github.com/grafana/grafana/blob/main/public/app/core/components/NavBar/NavBarItemMenu.tsx#L73-L77

![CleanShot 2022-07-26 at 09 29 52](https://user-images.githubusercontent.com/37156449/181096046-26f270a9-46aa-47df-a30d-2597efee9318.gif)

I tried fiddling with the z-index but no joy....

**Comment by zuchka** [comment]
see #52773

**Comment by ashharrison90** [comment]
currently implementing some new navigation designs - plan to tackle this as part of that 👍

**Comment by ashharrison90** [comment]
so as mentioned this is fixed as part of the new navigation experience we're implementing. you can check out these changes via the `topnav` feature toggle - there's still some bugs to fix, etc. as such, i'm going to close this as completed, and look out for the new navigation release soon™️ 🥳
